ImputationPipeline/ScoreBGE/ScoreBGE.wdl

```diff
@@ -9,6 +9,8 @@ workflow ScoreBGE {
         String basename
         File weights
         Array[String]? sample_names
+        Boolean score_haploid_as_diploid
+        Boolean use_emerge_weight_format = false
```

Comment:
again, I'd rather just have a tool to convert to required format

Reply:
Fixed